tests/foreman/api/test_fifi.py

```diff
@@ -0,0 +1,51 @@
+"""Test FiFi
+
+There is a test in Robottelo, tests/foreman/ui/test_remoteexecution.py, that tests
+Satellite part of FiFi (Receptor installation) as end-to-end as possible. However,
+it doesn't test cloud.redhat.com, we chose to use IQE framework to run tests that need to
+do that. This test fetches the FiFi test results from the IQE framework into
+Robottelo so Satellite QE's tooling (Report Portal, Polarion) can use it transparently
+
+
+:Requirement: RemoteExecution
+
+:CaseAutomation: Automated
+
+:CaseLevel: Component
+
+:CaseComponent: RemoteExecution
+
+:TestType: Functional
+
+:CaseImportance: High
+
+:Upstream: No
+"""
+import pytest
+import requests
+
+
+@pytest.mark.tier2
+def test_fifi():
+    """Fetch FiFi test result from IQE's Jenkins
+
+    :id: d63f87e5-66e6-488a-8c44-4129259494a6
+
+    :expectedresults: There are some tests found and all of the passed
+    """
+    job = requests.get(
+        'https://insights-stg-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/view/'
```

Comment:
@lhellebr we should add this urls to robottelo.properties files.

Reply:
our automation will replace it later with right url.

Reply:
Of course, I planned that and forgot. Will do :)

Reply:
Fixed in this PR. TODO: set this up in automation config.